### ISSUE TITLE
Controling volume using external soundcard

### ISSUE THREAD

**Issue by eddiematos** [olejon/spotcommander#94]
Hello. First off, thanks for this really, really great app.

Everything was working really well until I started using an external soundcard (Behringer UCA202) and now suddenly I can't control the volume from the android app. Everything else continues to work fine.

I'm using Lubuntu and can normally change the volume using a program called "amixer". I'm no expert, but I know that before using the external soundcard I could change the volume using the command *amixer set Master 50%* (to set the volume to 50% for example). Since plugging in the external soundcard now this command doesn't work and instead I have to use *amixer set PCM 50%*. Perhaps this is indicative of the change that occurred which means that the app can no longer talk with my soundcard.

**Comment by olejon** [comment]
You can just change the source code to use PCM instead.

Look at these lines:

https://github.com/olejon/spotcommander/blob/master/spotcommander/bin/spotcommander-remote#L163

https://github.com/olejon/spotcommander/blob/master/spotcommander/bin/spotcommander-remote#L257

Change Master to PCM and restart the daemon.

**Comment by olejon** [comment]
Closed due to no response.

**Comment by eddiematos** [comment]
Hello Ole. Sorry for the slow reply to your helpful comment. Unfortunately it seems I had not enabled reply notifications properly. Interestingly I found that the bug was due to the way in which I had configured ALSA. Indeed, your source code already uses "PCM" as the control to twiddle - though I don't know if maybe I'm now seeing a different version of your code. But regardless, my ALSA was not behaving properly because it configured correctly, and anyone who sees this and has a similar problem should take a look at this: https://bbs.archlinux.org/viewtopic.php?id=123250
